.tours/2---object agnostic design.tour

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "2 - Object-Agnostic Design",
+    "steps": [
+        {
+            "file": "es-base-code/main/default/objects/Customer_Fields__mdt/fields/Sobject_Type__c.field-meta.xml",
+            "description": "The app uses Custom Metadata Customer Fields to allow administrators to configure Lightning Web Components and Flow Screen Components to work with any object",
+            "line": 7
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/customerList/customerList.html",
+            "description": "The customerList and customerTile Lightning Web Components can display information from any object as long as the object names and fields are in the Custom Metadata Customer Fields",
+            "line": 14
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/customerList/customerList.js-meta.xml",
+            "description": "The customerList Lightning web component uses a design attribute to allow for users working in Lightning App Builder to control which object an instance of the Lightning Web Component should display",
+            "line": 11
+        },
+        {
+            "file": "es-space-mgmt/main/default/classes/ReservationManagerController.cls",
```

Comment:
Reorder this to go right after the metadata type step.. Ideally I would see the flow as - Create Metadata type for config.. Create Object Agnostic Apex Class, and use it to create object agnostic component. And finally configure the component using App Builder.

Reply:
done!